mne/time_frequency/_stockwell.py

```diff
@@ -103,7 +103,13 @@ def tfr_array_stockwell(data, sfreq, fmin=None, fmax=None, n_fft=None,
                         width=1.0, decim=1, return_itc=False, n_jobs=1):
     """Compute power and intertrial coherence using Stockwell (S) transform.
 
-    See [1]_, [2]_, [3]_, [4]_ for more information.
+    Same computation as the function tfr_stockwell.
+    Difference: tfr_stockwell takes epochs in the mne.Epochs format as input.
+    This function, tfr_array_stockwell, takes epochs in the numpy.array format
+    as input.
```

Comment:
```suggestion
    Same computation as `~mne.time_frequency.tfr_stockwell`, but operates on
    :class:`NumPy arrays <numpy.ndarray>` instead of `~mne.Epochs` objects.
```